### DISCUSSION TITLE
Is it possible to copy both timestamps at once (as one phrase, separated by space) or quickly run an FFmpeg command with both timestamps?

### DISCUSSION THREAD

**Discussion by YousufSSyed** [mifi/lossless-cut#1526] [Q&A] [answered]
What I want to do is get two exact frame timestamps with either LosslessCut or another FFmpeg GUI, and use them in my own FFmpeg command, ideally something faster than having to copy both times into a text editor or something like that. The issue with Losslesscut is that it only lossessly trim files, I wan to hardsub and re-encode them. I wonder if its possible to copy both timestamps at once and have a phrase like this "00:30.00 00:45.20", or somehow quickly have both times used in an FFmpeg command, like "ffmpeg -ss 00:30.00 -to 00:45.20 -i ..."?

**Comment by mifi** [comment, marked as answer]
Hi. You can look at the recent ffmpeg commands in the top menu

**Reply by YousufSSyed** [reply]
Thanks for that suggestion, though it would be great if I could copy the command without hitting export, so I made this feature request: https://github.com/mifi/lossless-cut/issues/1529.
